docs/DOCKER.md

```diff
@@ -35,7 +35,9 @@ Instructions
 Otherwise, to build an existing environment with the most updated DB, remove DB volume by running `docker volume rm anyway_db_data`.
 Note - this will delete all of your local DB data!
 
-**5.** Build anyway container with updated DB data (in anyway main directory): `docker-compose -f docker-compose.yml build --build-arg RESTORE_DB=TRUE --build-arg GDRIVE_FILE_ID=<GDRIVE_FILE_ID value>`
+**5.** Build anyway container with updated DB data (in anyway main directory): `docker-compose -f docker-compose.yml build --build-arg RESTORE_DB=TRUE --build-arg GDRIVE_FILE_ID="<GDRIVE_FILE_ID value>"`
+*<GDRIVE_FILE_ID value> - see part 3
+*If you're having this kind of ERROR:Need service name for --build-arg option, make sure the docker-compose version is 1.25.5 and above(check with docker-compose --version)
```

Comment:
@yossich6 from a quick look in [docker/compose repo](https://github.com/docker/compose/commit/583c673c8a1cdf5473c93d28f95433fffec55daf) it seems that version 1.25.2 is enough (I use 1.25.4 and it works well).
regarding the command docker-compose --version - please add ` to it (so it will be in the form of code)